lib/aws/xray/error_handlers.rb

```diff
@@ -26,11 +26,19 @@ class ErrorHandlerWithSentry
       ERROR_LEVEL = 'warning'.freeze
 
       def call(error, payload, host:, port:)
-        ::Raven.capture_exception(
-          error,
-          level: ERROR_LEVEL,
-          extra: { 'payload' => payload, 'payload_raw' => payload.unpack('H*').first }
-        )
+        if defined?(::Sentry)
+          ::Sentry.capture_exception(
+            error,
+            level: ERROR_LEVEL,
+            extra: { 'payload' => payload, 'payload_raw' => payload.unpack('H*').first }
+          )
+        elsif defined?(::Raven)
+          ::Raven.capture_exception(
+            error,
+            level: ERROR_LEVEL,
+            extra: { 'payload' => payload, 'payload_raw' => payload.unpack('H*').first }
+          )
+        end
```

Comment:
If neither sentry-raven nor sentry-ruby is set, this method silently finishes, so I think it should raise an error. This method is defined on the ErrorHandlerWith**Sentry** class, so it's supposed to require either sentry gem.

Reply:
Added checking the existence of gems in initialization, and make it raise error when #call:  https://github.com/cookpad/aws-xray/pull/102/commits/7d9050ccf4bec5a0180e79aef2b0b9a7902c2dd8